rotkehlchen/tests/unit/decoders/test_aerodrome.py

```diff
@@ -283,3 +283,250 @@ def test_remove_liquidity(base_accounts, base_transaction_decoder, load_global_c
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr
+@pytest.mark.parametrize('base_accounts', [['0x123509D7e9e6576263B10100cf7EB016C64F73Ce']])
+def test_unlock_aero(base_accounts, base_transaction_decoder):
+    user_address, evmhash = base_accounts[0], deserialize_evm_tx_hash('0xb4166c9b0c6076197ab2c17bdef8a55b050880d7005d874152a9f23ce7626790')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=base_transaction_decoder.evm_inquirer,
+        tx_hash=evmhash,
+    )
+    expected_events = [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=(timestamp := TimestampMS(1740994131000)),
+            location=Location.BASE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            amount=FVal(gas_str := '0.000002416100602324'),
+            location_label=user_address,
+            counterparty=CPT_GAS,
+            notes=f'Burn {gas_str} ETH for gas',
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=140,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=Asset('eip155:8453/erc721:0xeBf418Fe2512e7E6bd9b87a8F0f294aCDC67e6B4/71294'),
+            amount=ONE,
```

Comment:
why is the amount One here instead of thw withdrawn amount?

Reply:
this is the nft being burnt(informational), the next event is the actual token being sent back to the user.